packages/chain-adapters/src/cosmossdk/CosmosSdkBaseAdapter.ts

```diff
@@ -0,0 +1,210 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+// no-unused-vars is temporarily disabled until more functions are implemented
+import { CAIP2, caip2, caip19 } from '@shapeshiftoss/caip'
+import { AssetNamespace } from '@shapeshiftoss/caip/dist/caip19/caip19'
```

Comment:
Seeing several conversations as resolved with no fix or explanation regarding them

Reply:
All the caip-related comments should be moved to #398 and will be addressed there.

Reply:
That is already merged. Guess it can be cleaned up more in following prs.